engineering/repositories.md

```diff
@@ -1,6 +1,16 @@
-# Project Maintainers
+# Repository Guidelines
 
-## General Guidelines
+## Repository creation request
+
+In order to create a new repository, this should be requested through a [infrastructure issue](https://github.com/src-d/issues-infrastructure), and will be handled by @mcuadros.
+
+The name should be discussed with product, and the tags and description shuld be provided.
+
+## Repository privileges
+
+Each repository should belong to a team, and the full team has admin rights over this repository. If the repository engineering wise, the admin will be @smola and @mcuadros. 
```

Comment:
Just double-checking: we really mean `admin` here, not `write`, right?

Reply:
Yep I mean admin, is the rights I gave, as soon we have the terraform configuration for GitHub, it should be write